### PR TITLE
Task 3146

### PR DESCRIPTION
**Pull request by Myrthan**
Test implementation
https://github.com/facebook/presto/issues/3145

### CODE REVIEW

**Comment by nileema** [comment]
Can you add a description for what the PR does and a better commit message?

File: presto-main/src/main/java/com/facebook/presto/sql/planner/LogicalPlanner.java [hunk 1]
```diff
@@ -24,11 +24,7 @@
 import com.facebook.presto.sql.analyzer.Field;
 import com.facebook.presto.sql.analyzer.TupleDescriptor;
 import com.facebook.presto.sql.planner.optimizations.PlanOptimizer;
-import com.facebook.presto.sql.planner.plan.DeleteNode;
-import com.facebook.presto.sql.planner.plan.OutputNode;
-import com.facebook.presto.sql.planner.plan.PlanNode;
-import com.facebook.presto.sql.planner.plan.TableCommitNode;
-import com.facebook.presto.sql.planner.plan.TableWriterNode;
+import com.facebook.presto.sql.planner.plan.*;
```
Comment (zinking) [inline presto-main/src/main/java/com/facebook/presto/sql/planner/LogicalPlanner.java]: usually we do the reverse.

File: presto-parser/src/main/java/com/facebook/presto/sql/parser/AstBuilder.java [hunk 2]
```diff
@@ -147,7 +147,8 @@ public Node visitUse(@NotNull SqlBaseParser.UseContext context)
     @Override
     public Node visitCreateTableAsSelect(@NotNull SqlBaseParser.CreateTableAsSelectContext context)
     {
-        return new CreateTableAsSelect(getQualifiedName(context.qualifiedName()), (Query) visit(context.query()));
+        boolean createTableWithoutData = context.query().withOrWithoutData() != null ? context.query().withOrWithoutData().NO() != null : false;
```
Comment (zinking) [inline presto-parser/src/main/java/com/facebook/presto/sql/parser/AstBuilder.java]: will it be better to extract `context.query().withOrWithoutData()` into an expression? save some calls.

**Comment by KBP-TDC** [comment]
@Myrthan : are you actively working on this? If not, I know someone on the Teradata team who is ready to pick this up.

**Comment by cberner** [comment]
@Myrthan please re-open if you're still working on this

**Pull request closed without merging**